.github/workflows/codespell.yml

```diff
@@ -9,4 +9,5 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v1
       - run: pip install codespell
-      - run: codespell
+      # When using this Action in other repos, the --skip option below can be removed
+      - run: codespell --skip=./.*,./codespell_lib,./example/code.c
```

Comment:
This now skis `codespell_lib` which is where the library code actually lives, which is not so helpful. Maybe:
```
codespell --skip=./.*,./example/code.c,*.pyc,./codespell_lib/data,test_basic.py
```